README.md

```diff
@@ -1,13 +1,10 @@
-Kubernetes Custom Metrics Adapter for Prometheus
-================================================
+# Kubernetes Resource Metrics and Custom Metrics Adapter for Prometheus
```

Comment:
That's still a mouthful, but how about "Prometheus backed Kubernetes Metrics APIs", at least a bit shorter?